### ISSUE TITLE
error in de_novo_fit_penalty

### ISSUE THREAD

**Issue by conormcguinness9016** [AlexandrovLab/SigProfilerExtractorR#7]
Hi, I'm trying to use sigprofiler to extract mutations from mm10 catalogues.
I'm getting an error when I run the code:
Line:
> sigprofilerextractor("vcf", 
  "sigprofiler",
  "/Volumes/archive/cancergeneticslab/ConorM/MouseAnalysis/totalVCFhets/", 
  "mm10")
Error:

TypeError: sigProfilerExtractor() got an unexpected keyword argument 'de_novo_fit_penalty'

Other info: 
> sessionInfo()
R version 4.1.2 (2021-11-01)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Red Hat Enterprise Linux

> py_config()
python:         /usr/bin/python3.6
libpython:      /usr/lib64/libpython3.6m.so
pythonhome:     //usr://usr
version:        3.6.8 (default, Aug 13 2020, 07:46:32)  [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]
numpy:          /usr/local/lib64/python3.6/site-packages/numpy
numpy_version:  1.17.4
sys:            [builtin module]

NOTE: Python version was forced by use_python function

Trying to set de_novo_fit_penalty manually did not help. Any suggestions?
Thanks
Conor

**Comment by marcos-diazg** [comment]
Dear Conor,

Thanks so much for your question. Could you please check that you are using the latest SigProfilerExtractor version? You can do that by running `pip install SigProfilerExtractor --upgrade`. Also, you can try to install the tool on a specific conda environment by using the following commands:

```
conda create --name sigprofilerextractorR -y
conda activate sigprofilerextractorR
conda install -c conda-forge r-base r-devtools r-reticulate python=3.9 -y
echo 'devtools::install_github("AlexandrovLab/SigProfilerExtractorR")' | Rscript -
```

Please let me know if this helps. Best,

Marcos

**Comment by marcos-diazg** [comment]
Please reopen the ticket if you are still experiencing any issues. Thanks for your interest!

**Comment by frvirard** [comment]
Hi, I'm experiencing the same issue after having updated SigProfilerExtractor to v1.1.13

**Comment by marcos-diazg** [comment]
As indicated in #10, this has now been fixed in the most recent v1.1.13 of SigProfilerExtractorR. Thanks for letting us know, and sorry for the inconvenience.

**Comment by Giuseppe1995** [comment]
Hi! sorry If I'm taking back this issue, but I had a similar error with the version 1.1.16. I installed it with pip (`pip install SigProfilerExtractor`) and found a similar error, that is

> TypeError: sigProfilerExtractor() got an unexpected keyword argument 'refit_denovo_signatures'

Then, thanks to your comments, I tried to solve the problem by dowloading the v1.1.13 (`pip install SigProfilerExtractor==1.1.13`) and everything worked fine. I don't know if is a bug or something else but I just wanted to report this.
Thank you for your tool,
Giuseppe.
